config/flags/flags.go

```diff
@@ -307,6 +307,11 @@ var (
 		Usage: "Limit maximum size in some RPC calls execution",
 		Value: gossip.DefaultConfig(cachescale.Identity).MaxResponseSize,
 	}
+	StructLogLimitFlag = cli.IntFlag{
+		Name:  "rpc.structloglimit",
+		Usage: "Limit maximum number of debug logs for structured EVM logs",
```

Comment:
Can you add something about the interpretation of 0 and negative values?

Reply:
Done